contrib/commit-name.sh

```diff
@@ -8,5 +8,11 @@ gitref=${1:-HEAD}
 
 last_tag=$(git describe --tags --abbrev=0 "$gitref")
 ver=$(git show "$gitref:VERSION")
+major=$(echo $ver | cut -f 1 -d .)
+minor=$(echo $ver | cut -f 2 -d .)
 nb=$(git rev-list --count "$gitref" "^$last_tag")
-echo "$ver+$nb"
+if [ $major = 0 -a $minor -le 5 ]; then
+  echo "$ver+$nb"
+else
+  echo "$ver.$nb"
```

Comment:
Ah yes, should be just `echo $ver` if `$nb` is zero, I guess. Good opportunity to fix; will do next week.

Maybe we could eventually rewrite these scripts in a nicer programming language, like, say, Julia.